server/repository/src/migrations/v1_06_00/mod.rs

```diff
@@ -96,352 +41,16 @@ impl Migration for V1_06_00 {
 async fn migration_1_06_00() {
     use crate::migrations::*;
     use crate::test_db::*;
-    use chrono::NaiveDateTime;
-    use diesel::prelude::*;
-    use item_link::dsl as item_link_dsl;
-    use requisition_line::dsl as requisition_line_dsl;
-    use stock_movement::dsl as stock_movement_dsl;
-    use stock_on_hand::dsl as stock_on_hand_dsl;
-
-    table! {
-        item_link {
-            id->Text,
-            item_id->Text,
-        }
-    }
-    #[derive(Queryable, Debug, PartialEq)]
-    struct ItemLinkRow {
-        id: String,
-        item_id: String,
-    }
-
-    table! {
-        stock_on_hand(id) {
-            id -> Text,
-            item_id -> Text,
-            store_id -> Text,
-            available_stock_on_hand -> BigInt,
-        }
-    }
-    #[derive(Queryable, Debug, PartialEq)]
-    struct StockOnHandRow {
-        id: String,
-        item_id: String,
-        store_id: String,
-        available_stock_on_hand: i64,
-    }
-
-    table! {
-        stock_movement (id) {
-            id -> Text,
-            item_id -> Text,
-            store_id -> Text,
-            quantity -> BigInt,
-            datetime -> Timestamp,
-        }
-    }
-    #[derive(Queryable, Debug, PartialEq)]
-    struct StockMovementRow {
-        id: String,
-        item_id: String,
-        store_id: String,
-        quantity: i64,
-        datetime: NaiveDateTime,
-    }
-
-    table! {
-        requisition_line (id) {
-            id -> Text,
-            requisition_id -> Text,
-            item_link_id -> Text,
-            requested_quantity -> Integer,
-            suggested_quantity -> Integer,
-            supply_quantity -> Integer,
-            available_stock_on_hand -> Integer ,
-            average_monthly_consumption -> Integer,
-            snapshot_datetime -> Nullable<Timestamp>,
-            approved_quantity -> Integer,
-            approval_comment -> Nullable<Text>,
-            comment -> Nullable<Text>,
-        }
-    }
-
-    #[derive(Queryable, Debug, PartialEq, Default)]
-    struct RequisitionLineRow {
-        id: String,
-        requisition_id: String,
-        item_link_id: String,
-        requested_quantity: i32,
-        suggested_quantity: i32,
-        supply_quantity: i32,
-        available_stock_on_hand: i32,
-        average_monthly_consumption: i32,
-        snapshot_datetime: Option<NaiveDateTime>,
-        approved_quantity: i32,
-        approval_comment: Option<String>,
-        comment: Option<String>,
-    }
 
-    let previous_version = v1_05_00::V1_05_00.version();
     let version = V1_06_00.version();
 
+    // This test allows checking sql syntax
     let SetupResult { connection, .. } = setup_test(SetupOption {
-        db_name: &format!("migration_{version}_aye"),
```

Comment:
Hahaha whoops this was left over from a time I was trying to find the right darn sqlite file.

Reply:
Forgot, I moved the old test into a new merge migration related test and added the "default" test for 1.7.0... Only guessed that the test was fully merge related so I thought this will make it clear...